itests/kit/ensemble.go

```diff
@@ -172,6 +173,7 @@ func NewEnsemble(t *testing.T, opts ...EnsembleOpt) *Ensemble {
 	}
 
 	build.EquivocationDelaySecs = 0
```

Comment:
Why are we keeping this in both build and build constants?

Reply:
A full cleanup of all deprecations is out-of-scope of this PR, but yes this one is also easy so I will include it.